server/mergin/sync/permissions.py

```diff
@@ -158,11 +158,25 @@ def get_user_project_role(
         return None
 
 
-def require_project(ws, project_name, permission):
-    workspace = current_app.ws_handler.get_by_name(ws)
+def is_active_workspace(workspace):
```

Comment:
do we need this function at all? there is already similar check https://github.com/MerginMaps/server/blob/gh%232381-prevent-download-anonymous-inactive-ws/server/mergin/sync/workspace.py#L68

we would only need to handle current_user properly

Reply:
I am not sure, this just abstracted code from require_project and required_project_by_uuid .

Reply:
I am only worried we use various similar functions for doing the same thing and most probably not used everywhere - we might want to revisit this at some point.

Reply:
Thanks :) But currently I am no table to do some changes there .